source/INIT.sqf

```diff
@@ -98,14 +98,14 @@ if (DedicatedMission) then {
     diag_log format ["server, selected CP: %d", commandpointsblu1];
 } else {
     DUWSMP_CP_death_cost = 1; //first element
-    debugmode = false;
+    debugmode;

```

Comment:
just remove this line, it is not needed since debugmode is set to false earlier and having just the variable with a semicolon does nothing but take space.